### DISCUSSION TITLE
"current location" text in widget

### DISCUSSION THREAD

**Discussion by Araxeus** [breezy-weather/breezy-weather#639] [General]
![Screenshot_2024-01-23-16-13-27-263_com.mi.android.globallauncher-edit.jpg](https://github.com/breezy-weather/breezy-weather/assets/78568641/164ce07f-2938-4a70-b9a1-f38004eb6b24)

Why does it say that? And how can I remove this useless text?

Also could we replace it with text describing the current weather like the stock weather app?

![Screenshot_2024-01-23-16-16-51-739_com.mi.android.globallauncher-edit.jpg](https://github.com/breezy-weather/breezy-weather/assets/78568641/2e0ec25f-e3a6-443b-991e-2acd0338ec0a)

**Comment by papjul** [comment]
This is the name of your location.
Since you chose a weather source without reverse geocoding, we fallback to displaying "Current location".
If it is an important matter to you, and you want to avoid that, you can either:
- Add your location manually from search and not rely on GPS
- Use a different weather source with reverse geocoding support: https://github.com/breezy-weather/breezy-weather/blob/main/docs/SOURCES.md

Widgets are not based on AOSP weather app since there is no weather app in AOSP. Widgets are provided as they were in Geometric Weather with only major bug fixes. If a major contributor wants to step in to help, the whole Widgets part needs an important redesign (specs and some mockups are already done).
